packages/lab/src/calendar/useCalendar.ts

```diff
@@ -135,7 +135,7 @@ export function useCalendar(props: useCalendarProps) {
   const [calendarFocused, setCalendarFocused] = useState(false);
 
   const [focusedDate, setFocusedDateState] = useState<DateValue>(
-    startOfMonth(visibleMonth)
+    (selectedDate as DateValue) ?? today ?? startOfMonth(visibleMonth)
```

Comment:
minor thing, logic here suggests today is also a preference, but not covered in test?

Reply:
Is this logic right? 

- We are using the controlled selectedDate which would stop this working if it was an uncontrolled calendar.
- `today` is the function to be called from internationalized which will never be undefined. This currently works because react calls it due to the lazy useState initialisation.
- startOfMonth will never be called because today is always valid.

There are now a lot of cases that need to be tested:
- Default, Selected dates receive initial focus
- Once navigating, the start of the month will always receive the initial focus
- If the visibleMonth does not contain today or selected date the first date receives focus (this is only happening right now due to the useEffect below)

Reply:
@Fercas123 if useful, the [highlighted dates](https://113582bd.saltdesignsystem-storybook.pages.dev/?path=/story/lab-calendar--highlighted-dates) example is having styling issues when the highlight is in different month. .